lua/lspsaga/diagnostic.lua

```diff
@@ -184,6 +184,15 @@ M.navigate = function(direction)
   end
 end
 
+M.toggle_virtual_text = function()
+  if config.use_diagnostic_virtual_text == true then
+    config.use_diagnostic_virtual_text = false
+  else
+    config.use_diagnostic_virtual_text = true
+  end
+  vim.diagnostic.config({virtual_text = not config.use_diagnostic_virtual_text})
```

Comment:
```suggestion
  vim.diagnostic.config({virtual_text = config.use_diagnostic_virtual_text})
```

nvm, I meant that adding not wouldn't require line 188, sorry about that